core/src/test/kotlin/io/bkbn/kompendium/core/KompendiumAuthenticationTest.kt

```diff
@@ -92,14 +93,18 @@ class KompendiumAuthenticationTest : DescribeSpec({
       ) { customAuthConfig() }
     }
     it("Can provide multiple authentication strategies") {
+      data class TestAppPrincipal(val key: String) : Principal
```

Comment:
new warning here for the now deprecated `Principal`. Ideally either the third party library is updated, or a new solution for api key handling is implemented.

Reply:
hmm ok we won't worry about this warning for now